recipe/meta.yaml

```
@@ -25,16 +25,16 @@ source:
       - support_nvtx3_win.diff        # [win64]
 
 build:
-  number: 3
-  # TODO: turn on win64 + CUDA 12 once it's ready
-  skip: true  # [cuda_compiler_version not in ("11.8", "12.0") or (cuda_compiler_version == "12.0" and win64)]
+  number: 4
+  skip: true  # [cuda_compiler_version not in ("11.8", "12.0")]
   script:
     # CuPy default detects CUDA from nvcc, but on Conda-Forge's dockers nvcc lives in a different place...
     # With conda-forge/nvcc-feedstock#58, CUDA_PATH is set correctly
     - export NVCC="$(which nvcc)"                                                # [linux]
     {% if cuda_major >= 12 %}
     - export CUPY_NUM_BUILD_JOBS=1                                               # [(aarch64 or ppc64le) and (build_platform == target_platform)]
     - export CUDA_PATH=$PREFIX/targets/{{ target_name }}                         # [linux]
+    - set "CUDA_PATH=%PREFIX%"                                                  # [win]
```

Think we may want `%LIBRARY_PREFIX%` on Windows, which points to `%PREFIX%\Library` that contains headers and libraries

```suggestion
    - export CUDA_PATH=$PREFIX/targets/{{ target_name }}                         # [linux]
    - set "CUDA_PATH=%LIBRARY_PREFIX%"                                           # [win]
```

---

I find this confusing. According to conda-build docs `LIBRARY_PREFIX` is under the build prefix (which we need), but it seems to be under host:
```
"nvcc is D:\\bld\\cupy_1699021281200\\_build_env\\Library\\bin\\nvcc.exe --use-local-env, CUDA path is %PREFIX%\Library"
```
and
```
  **************************************************
  *** WARNING: nvcc path != CUDA_PATH
  *** WARNING: nvcc path: D:\bld\cupy_1699021281200\_build_env\Library
  *** WARNING: CUDA_PATH: D:\bld\cupy_1699021281200\_h_env\Library
  **************************************************
```

---

Clarifying with PR: https://github.com/conda/conda-build/pull/5065